server/channels/app/limits_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetUserLimits(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	t.Run("base case", func(t *testing.T) {
+		userLimits, appErr := th.App.GetUserLimits()
+		require.Nil(t, appErr)
+
+		// InitBasic creates 3 users by default
+		require.Equal(t, int64(3), userLimits.ActiveUserCount)
+		require.Equal(t, int64(10000), userLimits.MaxUsersLimit)
+		require.Equal(t, int64(500), userLimits.LowerBandUserLimit)
+		require.Equal(t, int64(9000), userLimits.UpperBandUserLimit)
```

Comment:
These are a bit brittle when we change the values of the constants. We may want to use the constants, but 0/5.